windows-apps-src/xbox-live/using-xbox-live/best-practices/fine-grained-rate-limiting.md

```diff
@@ -90,30 +90,52 @@ When the associated user and title count is at or above either the burst or sust
 
 ## Implemented limits
 
-The following services will be implementing FGRL limits which will **begin enforcing in May 2016**. To reiterate, these limits will be the same across all sandboxes and titles. **Any title that is currently in XDP or Dev Center and shipping around May 2016 will be considered Legacy and therefore exempted.** The limits below are not finalized yet
-
-| **Service Name**          | **Burst Limit (15 seconds per user per title)** | **Sustain Limit (300 seconds per user per title)** |
-|---------------------------|-------------------------------------------------|----------------------------------------------------|
-| Game Clips Transfer       | 50                                              | 300                                                |
-| Game Clips Metadata       | 50                                              | 300                                                |
-| Smart Match (not MPSD)    | 10                                              | 100                                                |
-| Title Storage             | 30                                              | 300                                                |
-| Stats Read (not creation) | 100                                             | 300                                                |
-| Achievements Read         | 100                                             | 100                                                |
-| Leaderboards Read         | 30                                              | 100                                                |
-| Profile                   | 10                                              | 30                                                 |
-| People                    | 10                                              | 30                                                 |
-| Privacy                   | 10                                              | 30                                                 |
-| Presence                  | Read 10, Write 3                                | Read 100, Write 30                                 |
+The following services will have implemented FGRL limits, with enforcement of these limits in place since **May 2016**. To reiterate, these limits will be the same across all sandboxes and titles. **Any title that is was published via XDP or Dev Center and shipped prior to May 2016 will be considered Legacy and therefore exempted.**
```

Comment:
Typo: "Any title that is was" remove the extraneous "is".

Also, change "will be considered Legacy" to "is considered legacy"